client/my-sites/stats/geochart/index.jsx

```diff
@@ -37,29 +66,31 @@ module.exports = React.createClass( {
 		}
 	},
 
-	resize: function() {
+	resize() {
 		if ( this.visualizationsLoaded ) {
 			this.drawData();
 		}
 	},
 
-	drawData: function(){
-		var regionCodes = [];
-
-		if( this.props.data.length > 1 ) {
-			var data = window.google.visualization.arrayToDataTable( this.props.data ),
-				node = this.refs.chart,
-				width = node.clientWidth,
-				options = {
-					width: 100 + '%',
-					height: width <= 480 ? '238' : '480',
-					keepAspectRatio: true,
-					enableRegionInteractivity: true,
-					region: 'world',
-					colorAxis: { colors: [ '#FFF088', '#F34605' ] }
-				};
-
-			this.props.dataList.response.data.map( function( country ) {
+	drawData() {
+		const regionCodes = [];
+		const { data, dataList } = this.props;
+
+		if ( data.length ) {
+			const chartData = window.google.visualization.arrayToDataTable( data );
+			const node = this.refs.chart;
+			const width = node.clientWidth;
+
+			const options = {
+				width: 100 + '%',
+				height: width <= 480 ? '238' : '480',
+				keepAspectRatio: true,
+				enableRegionInteractivity: true,
+				region: 'world',
+				colorAxis: { colors: [ '#FFF088', '#F34605' ] }
+			};
+
+			dataList.response.data.map( function( country ) {
 				if ( -1 === regionCodes.indexOf( country.region ) ) {
```

Comment:
Yes - if all countries share the same region code, we pass along the region code to have the map zoomed in.